siriuspy/siriuspy/devices/fofb_acq.py

```diff
@@ -894,6 +970,30 @@ def check_prbs_bpmposy_levels(self, level0, level1=None):
                 return False
         return True
 
+    def wait_prbs_bpmposy_levels(
+            self, level0, level1=None, timeout=_FamFOFBAcqBase.DEF_TIMEOUT):
+        """
+        Wait for BPM PosY PRBS excitation levels to be equal to
+        desired values.
+
+        Args:
+            level0 (numpy.ndarray, (NR_BPM,)):
+                desired values for BPM PosY PRBS excitation level 0.
+            level1 (optional, numpy.ndarray, (NR_BPM,)):
+                If None, we consider level1 = -level0. Defaults to None.
+
+        Returns:
+            bool: indicate whether levels 0 and 1 of BPM PosY PRBS
+                excitation are in desired values.
+
+        """
```

Comment:
To avoid repetition, do you think this docstring could simply tell the user to refer to the wait_prbs_bpmposx_levels function?

Reply:
I added a commit changing the `wait_...` methods docstrings to refer to respective `check_...` methods.